### ISSUE TITLE
Question: Why is the build RCP with JBR removed?

### ISSUE THREAD

**Issue by amirmv2006** [JetBrains/fbme#1]
Hi, I have learned a lot about how to setup my own MPS language from your project here, especially for CI/CD, I copied a lot of scripts from here to my own project. However, recently I noticed the support for building RCP with JBR was removed. The main advantage of previous solution IMHO was the fact that the IDE would not depend on the java that is installed in the user's machine. Is that not valid anymore or there's an alternative I'm missing?

**Comment by qradimir** [comment]
Hello, @amirmv2006! It's pleasant to hear that this project helps with your tasks! 

Regarding the RCP-with-JBR build, it was indeed dropped recently with no viable alternative. The main reason for that is that we've noticed failures on the RCP's startup with bundled JBR. I was unable to figure out what was the cause of it, so we decided to switch to regular RCP builds with no JRE bundled. RCPs with no bundled JRE indeed have a drawback that it depends on JRE installation on user machine. However, it is fine for us at the moment.

In future, we have plans to restore builds with bundled JBR, but I am not sure how it will be configured taking into account that the project build recently integrated with TeamCity.
